app/code/Magento/Catalog/Test/Mftf/Test/AdminBackorderAllowedAddProductToCartTest.xml

```diff
@@ -31,6 +31,7 @@
             <!-- Set Magento back to default configuration -->
             <magentoCLI command="config:set {{CatalogInventoryOptionsShowOutOfStockDisable.path}} {{CatalogInventoryOptionsShowOutOfStockDisable.value}}" stepKey="setConfigShowOutOfStockFalse"/>
             <magentoCLI command="config:set {{CatalogInventoryItemOptionsBackordersDisable.path}} {{CatalogInventoryItemOptionsBackordersDisable.value}}" stepKey="setConfigAllowBackordersFalse"/>
+            <magentoCLI command="cache:flush" stepKey="flushCache"/>
```

Comment:
@engcom-Echo can we flush only config cache here?

Reply:
@ihor-sviziev command="cache:flush config"

Reply:
I believe it could be replaced to:
```suggestion
            <magentoCLI command="cache:clean config full_page" stepKey="cleanInvalidatedCache/>
```
Similar to this:
https://github.com/magento/magento2/blob/a3c2af0498d21e3f689bc8dd690d7c633a025276/app/code/Magento/Backend/Test/Mftf/Test/AdminMenuNavigationWithSecretKeysTest.xml#L27-L28